src/metric/SampleMetricManager.py

```diff
@@ -89,6 +89,16 @@ def get_metric_instances(self) -> list[SampleMetric.SampleMetric]:
         """
         return list(self._sample_metrics.values())
 
+    def get_metric_indices(self) -> pd.Index:
+        """
+        Returns the indices where one or more sample metrics
+        have been calculated.
+
```

Comment:
"one or more"

Does this refer to the subclass-imposed dict, or simply that there may be multiple indices?

If the former is true, it should probably not be listed here since that behavior should not be an intrinsic part of the architecture